pkg/controllers/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -79,21 +81,30 @@ func NewConfigObserver(
 			configobserver.WithPrefix(o, configobservation.OAuthServerConfigPrefix))
 	}
 
+	listers := configobservation.Listers{
+		ConfigMapLister: kubeInformersForNamespaces.ConfigMapLister(),
+		SecretsLister:   kubeInformersForNamespaces.SecretLister(),
+		IngressLister:   configInformer.Config().V1().Ingresses().Lister(),
+
+		APIServerLister_:     configInformer.Config().V1().APIServers().Lister(),
+		ClusterVersionLister: configInformer.Config().V1().ClusterVersions().Lister(),
+		InfrastructureLister: configInformer.Config().V1().Infrastructures().Lister(),
+		OAuthLister_:         configInformer.Config().V1().OAuths().Lister(),
+		ResourceSync:         resourceSyncer,
+		PreRunCachesSynced:   preRunCacheSynced,
+	}
+
+	// Check if the Console capability is enabled on the cluster and sync and add its informer and lister.
+	if enabledClusterCapabilities.Has("Console") {
+		preRunCacheSynced = append(preRunCacheSynced, configInformer.Config().V1().Consoles().Informer().HasSynced)
```

Comment:
I think this should be `listers.PreRunCachesSynced = append(listers.PreRunCachesSynced, ...` to make sure the above struct contains the right slice